install_prerequisites.bash

```diff
@@ -4,6 +4,9 @@ set -o errexit
 
 apt-get install -y puppet librarian-puppet
 
+# Needed to use the Docker upstream apt repositories.
+apt-get install apt-transport-https
```

Comment:
@nuclearsandwich: did you have a particular reason for not adding `-y` here? The other `apt-get install` lines in this file do have `-y`, so this one seems to deviate from that pattern.

Reply:
Nope. I tend to enable the global autoconfirm and so fail to notice when I forget `-y`. #42 looks good and is merged. Thanks!

Reply:
> I tend to enable the global autoconfirm

Would that be something to document?

Reply:
> Would that be something to document?

I think it's more reasonable that I stop doing it on shared production systems. IMO it's not a default to enforce on everyone.